linkerd/app/outbound/src/http/peer_proxy_errors.rs

```diff
@@ -24,14 +20,14 @@ impl<N> PeerProxyErrors<N> {
         Self { inner }
     }
 
-    pub fn layer() -> impl layer::Layer<N, Service = Self> + Clone + Copy {
-        layer::mk(Self::new)
+    pub fn layer() -> impl svc::layer::Layer<N, Service = Self> + Clone + Copy {
+        svc::layer::mk(Self::new)
     }
 }
 
-impl<T, N> NewService<T> for PeerProxyErrors<N>
```

Comment:
similarly, elsewhere in the code we tend to use `svc::NewService/Service`. The `svc` prefix isn't too verbose. I think we mostly do this because we used to have more structs named `Service` and `NewService`. This isn't so prevalent anymore but we still tend to do this elsewhere.